mmcv/transforms/utils.py

```diff
@@ -0,0 +1,132 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+
+import functools
+import inspect
+import weakref
+from collections import defaultdict
+from collections.abc import Iterable
+from contextlib import contextmanager
+from typing import Callable, Union
+
+from .base import BaseTransform
+
+
+class cacheable_method:
+
+    def __init__(self, func):
+
+        # Check `func` is to be bound as an instance method
```

Comment:
Check whether `func` is bound as an instance method

Reply:
We apply a non-strict check at L20. Is there any suggestion for a better way?